### DISCUSSION TITLE
Why the suggested-questions always return empty?

### DISCUSSION THREAD

**Discussion by chenlizhao** [langgenius/dify#4215] [General]
Hi, I just doployed the latest dify version on my computer, and I created a chat app, but when I enabled suggested next questions, the api call of http://10.200.5.50:81/console/api/installed-apps/3b5c1ad7-55be-4dfb-8fe7-224638d3735b/messages/deda0002-6554-45d5-89e5-48efa7259101/suggested-questions always return {"data": []}, is  there something missed by me？I have uploaded a document as knowledge base.

**Comment by superbread93a** [comment]
你解决了吗

**Comment by crazywoola** [comment]
You need to turn on the switch in the UI console.
